packages/core/src/base/body.scss

```diff
@@ -37,18 +37,5 @@ Style guide: style.base
   @extend %ds-base;
   background-color: $color-background-inverse;
   color: $color-base-inverse;
-  a {
```

Comment:
Moved link styling to `packages/core/src/base/typography/link.scss` for better organization